ResearchKit/Common/ORKHelpers.m

```diff
@@ -251,6 +251,12 @@ void ORKEnableAutoLayoutForViews(NSArray *views) {
     return bundle;
 }
 
+NSBundle *ORKDefaultLocaleBundle() {
+    NSString * path = [ORKBundle() pathForResource:@"en" ofType:@"lproj"];
+    NSBundle * bundle = [NSBundle bundleWithPath:path];
```

Comment:
Like the idea to use `objectForInfoDictionaryKey:@"CFBundleDevelopmentRegion"`. But this is for RK's localization, we might want to use `ORKBundle` instead of `mainBundle`.